kubernetes-client/src/main/java/io/fabric8/kubernetes/client/CreateOnlyResourceOperations.java

```diff
@@ -13,18 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+package io.fabric8.kubernetes.client;
 
-package io.fabric8.openshift.client.dsl;
-
-import io.fabric8.kubernetes.api.model.Doneable;
 import io.fabric8.kubernetes.client.dsl.Createable;
-import io.fabric8.openshift.api.model.LocalSubjectAccessReview;
-import io.fabric8.openshift.api.model.LocalSubjectAccessReviewFluentImpl;
-import io.fabric8.openshift.api.model.SubjectAccessReviewResponse;
 
-public abstract class CreateableLocalSubjectAccessReview extends
-  LocalSubjectAccessReviewFluentImpl<CreateableLocalSubjectAccessReview>
-  implements
-  Doneable<SubjectAccessReviewResponse>,
-  Createable<LocalSubjectAccessReview, SubjectAccessReviewResponse, CreateableLocalSubjectAccessReview> {
+public interface CreateOnlyResourceOperations<T, D> extends Createable<T, T, D> {
```

Comment:
This interface seems to be redundant now since it's only extending Creatable and not adding any methods. Why keep it?